doc/bibliography.bib

```diff
@@ -984,45 +994,47 @@ @Article{tironi95a
   doi       = {10.1063/1.469273},
 }
 
-@article{turner08a,
+@Article{turner08a,
   title={Simulation of chemical reaction equilibria by the reaction ensemble {M}onte {C}arlo method: {A} review},
-  author={Heath Turner, C. and Brennan, John K. and L\'{i}sal, Martin and Smith, William R. and Karl Johnson, J. and Gubbins, Keith E.},
+  author={Turner, C. Heath and Brennan, John K. and L{\'i}sal, Martin and Smith, William R. and Johnson, J. Karl and Gubbins, Keith E.},
   journal={Molecular Simulation},
   volume={34},
   number={2},
   pages={119--146},
   year={2008},
   doi={10.1080/08927020801986564},
-  publisher={Taylor \&{} Francis Group}
+  publisher={Taylor \& Francis Group}
 }
 
-@ARTICLE{tyagi07a,
-  author = {S. Tyagi and A. Arnold and C. Holm},
+@Article{tyagi07a,
+  author = {Tyagi, Sandeep and Arnold, Axel and Holm, Christian},
   title = {{ICMMM2D}: An accurate method to include planar dielectric interfaces via image charge summation},
   journal = {The Journal of Chemical Physics},
   year = {2007},
   volume = {127},
+  number = {15},
+  pages = {154723},
   doi = {10.1063/1.2790428},
-  pages = {154723}
 }
 
-@ARTICLE{tyagi08a,
+@Article{tyagi08a,
   author = {Sandeep Tyagi and Axel Arnold and Christian Holm},
-  title = {Electrostatic layer correction with image charges: A linear scaling method to treat slab 2{D} + h systems with dielectric interfaces},
+  title = {Electrostatic layer correction with image charges: {A} linear scaling method to treat slab 2{D} + h systems with dielectric interfaces},
```

Comment:
do you want to make the `h' into a math environment (or italic), as in the original title?

Reply:
Unfortunately, the BibTeX processor used by Sphinx currently doesn't handle math environments.